Documentation/Books/Manual/Views/ArangoSearch/DetailedOverview.md

```diff
@@ -118,6 +118,32 @@ of removing unused segments after release of internal resources.
   > reflected by queries invoked in subsequent ArangoDB transactions, while
   > in-progress ArangoDB transactions will still continue to return a
   > repeatable-read state.
+  
+  
+  ArangoSearch performs operations in its index based on numerous writer
+  objects that are mapped to processed segments. In order to control memory that
+  is used by these writers (in terms of "writers pool") one can use
+  `writebuffer*` properties of a view.
+
+- **writebufferIdle** (_optional_; type: `integer`; default: `64`;
+  to disable use: `0`)
+
+  Maximum number of writers (segments) cached in the pool.
+
+- **writebufferActive** (_optional_; type: `integer`; default: `0`;
+  to disable use: `0`)
+
+  Maximum number of concurrent active writers (segments) performs (a transaction).
```

Comment:
Why is `(a transaction)` in parentheses? It's not a full sentence without it.

Reply:
Fixed.